README.md

```diff
@@ -229,6 +231,21 @@ cfg_cmds.each do |cmd|
 end
 ```
 
+```ruby
+# Add static machine key at site level
+iis_config "MySite /commit:site /section:machineKey /validation:AES /validationKey:AAAAAA /decryptionKey:ZZZZZ" do
+  action :config
+end
```

Comment:
@philoserf definitely, for sample sake I decided to just leave the action even though it is unneeded.  Updated it to `:set` for all examples